doc/languages-frameworks/go.xml

```diff
@@ -24,7 +24,7 @@ deis = buildGoPackage rec {
     sha256 = "1qv9lxqx7m18029lj8cw3k7jngvxs4iciwrypdy0gd2nnghc68sw";
   };
 
-  goDeps = ./deps.json; <co xml:id='ex-buildGoPackage-3' />
+  goDeps = import ./deps.nix; <co xml:id='ex-buildGoPackage-3' />
```

Comment:
Is `import` necessary here? I don't see it included in the packages you converted from `deps.json`.